test/compose/cdi_device/tests.sh

```diff
@@ -0,0 +1,5 @@
+# -*- bash -*-
+
+ctr_name="cdi_device-test-1"
+
+podman exec "$ctr_name" sh -c 'test /dev/ksmg1'
```

Comment:
I run this locally this still passes if I change it to `/dev/ksmgblah` so this is not testing anything and must be fixed.

And if I change it to
```
podman exec "$ctr_name" ls /dev/
is "$output" "kmsg1" "check for kmsg1 from the cdi spec"
```
it still fails and see `ls: /dev/kmsg1: Permission denied` printed on stderr instead, not sure why as we run as root

Reply:
ouch, I forgot the `-e` or the `-c`.

In any case, I've added a bit more robust test, as it checks for the major and minor